File: Data Structures & Algorithms/two-integer-sum/submission-1.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int a,b,flag=0;
        for(int i=0;i<nums.size()-1;i++)
        {
            for(int j=1;j<nums.size();j++)
            {
                if(nums[i]+nums[j]==target and i!=j)
                {
                    a=i;
                    b=j;
                    flag=1;
                    break;
                }
            }
            if(flag) break;
        }
        return {a,b};
    }
};

File: Data Structures & Algorithms/two-integer-sum-ii/submission-1.py
class Solution:
    def twoSum(self, numbers: List[int], target: int) -> List[int]:
        i = 0
        j = len(numbers) - 1
        length = j

        for _ in range(length):
            if numbers[i] + numbers[j] > target:
                j -= 1
            elif numbers[i] + numbers[j] < target:
                i += 1
            else:  # numbers[i] + numbers[j] == target
                break

        return [i + 1, j + 1]